src/main/java/com/geronimo/config/security/jwt/filter/JwtAuthTokenFilter.java

```diff
@@ -32,9 +32,6 @@
     @Value("${jwt.header}")
     private String tokenHeaderName;
 
-    @Value("${jwt.scheme}")
-    private String scheme;
```

Comment:
Ok, I see it wasn't used in this filter, right?

Reply:
exactly

Reply:
I moved that stuff to JwtUtil, also removed this field from JwtAuthController :)

Reply:
cool